internal/api/middleware.go

```diff
@@ -112,7 +113,7 @@ func (a *API) limitEmailOrPhoneSentHandler() middlewareHandler {
 							emailRateLimitCounter.Add(
 								req.Context(),
 								1,
-								attribute.String("path", req.URL.Path),
+								metric.WithAttributes(attribute.String("path", req.URL.Path)),
```

Comment:
`WithAttributes` attaches  the attribute (in this case the URL Path) to the metric (in this case the email rate limit counter).  Running `attribute.String` as a standalone would create the attribute but it would not be linked to the `metric` / span

`WithAttributes` is syntactic sugar and [calls `WithAttributeSet` under the hood](https://github.com/open-telemetry/opentelemetry-go/blob/metric/v1.27.0/metric/instrument.go#L367)

Initially opted to use it as there's a single attribute - wasn't aware of the copy under the hood. `WithAttributeSet` would probably be better - thanks for flagging!